go.mod

```diff
@@ -47,8 +47,8 @@ require (
 	github.com/hashicorp/logutils v1.0.0
 	github.com/hashicorp/memberlist v0.1.5 // indirect
 	github.com/hashicorp/serf v0.8.5 // indirect
-	github.com/hashicorp/terraform v0.0.0
-	github.com/hashicorp/terraform-plugin-sdk v1.6.0
+	github.com/hashicorp/terraform v0.12.21
+	github.com/hashicorp/terraform-plugin-sdk v1.7.0
```

Comment:
these changes here we're auto-added by `go mod tidy` and are a noop.

Reply:
it doesn't like us using `v0.0.0` ?? because i would prefer that :P

Reply:
yep.. unfortunately `go mod tidy` inserts these changes automatically :( I could remove them here, but they'd pop up again next time...